wiki/upgrade-notes/drupal-11.md

```
@@ -2,128 +2,160 @@
+
+## What Changed
+
+### Versions
+- Drupal core: 9.4.5 → 11.3.9
+- PHP: 8.1 → 8.3
+- MariaDB: 10.3 → 10.6
+- Drush: 11 → 13
+- DDev type: drupal9 → drupal11
+
+### Modules Updated (Major Bumps)
+| Module | From | To |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)